src/core/components/layout/Container/Container.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import { Container, ContainerProps } from './Container';
+import { asChromaticStory, asPlayground } from '../../../../lib/story-intents';
+import { Story } from '../../../../@types/storybook-emotion-10-fixes';
+
+export default {
+	title: 'Source/src-layout/Container',
+	component: Container,
+	argTypes: {
+		border: {
+			control: {
+				disable: true,
+			},
+		},
+	},
```

Comment:
Can we disable the `cssOverrides` control as it doesn't really work in storybook

```suggestion
	argTypes: {
		border: {
			control: {
				disable: true,
			},
		},
		cssOverrides: {
			control: {
				disable: true,
			},
		},
	},
```

Reply:
As agreed offline, let's lift this up into the helper function rather than adding it everywhere